apps/src/aichat/types.ts

```diff
@@ -67,6 +68,13 @@ export enum Visibility {
   EDITABLE = 'editable',
 }
 
+export interface LevelAiChatSettings {
```

Comment:
We've been using lowercase in `aichat` for other names like `AichatLevelProperties`. Should we be consistent and use `LevelAichatSettings`?

Reply:
Will add a comment - `LevelAiCustomizations` will go away in the following change (it's here for now since the level editor files still reference it)

Reply:
> Will add a comment - LevelAiCustomizations will go away in the following change (it's here for now since the level editor files still reference it)

Oh cool! I was wondering about that. Thanks so much for the upcoming refactor on the levelbuilder end.